apps/website/src/pages/docs/textarea.mdx

```diff
@@ -9,20 +9,45 @@ group: inputs
 
 import PropsTable from '~/components/PropsTable.astro';
 import LiveExample from '~/components/LiveExample.astro';
-import Placeholder from '~/components/Placeholder.astro';
 import * as AllExamples from '~/examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='input-textarea' />
-
 <LiveExample src='Textarea.main.tsx' truncate={false}>
   <AllExamples.TextareaMainExample client:load />
 </LiveExample>
 
 Text areas (or multi-line fields) are larger text fields that can accommodate more text. It is vertically resizable and wraps text to the next line when maximum line length is reached. It automatically scrolls down to allow more text to be input when the bottom is reached. This type of text field implies that a larger input of text is possible and encouraged.
 
-It used to be possible to freely resize the text area horizontally in addition to vertically; this pattern is now deprecated, and text areas only allow vertical resizing for better predictability. The resize handle is optional; it may be omitted if resizing of the text area is not desired.
+<LiveExample src='Textarea.scroll.tsx'>
+  <AllExamples.TextareaScrollExample client:load />
+</LiveExample>
+
+If the area is shrunk to a size smaller than the amount of text within, a scrollbar appears to keep navigation possible. This behavior is especially important for text areas without a resizing handle, to avoid making text editing tedious.
+
+## Variants
+
+### Status
+
+<LiveExample src='Textarea.status.tsx'>
+  <AllExamples.TextareaStatusExample client:load />
+</LiveExample>
+
+If status messages are required, the styling is similar to [inputs](input). For the default textarea, the bottom border gains the color of the alert level, as well as the corresponding icon and message.
```

Comment:
This comment is just a reminder about "textarea" vs "text area".

Reply:
Updated "textarea" to be "text area"